setup.py

```diff
@@ -1,18 +1,17 @@
 #!/usr/bin/env python3
-
-# Start ignoring PyImportSortBear as imports below may yield syntax errors
-from bears import assert_supported_version
```

Comment:
In [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) it's mentioned that your import order should be

* standard library imports
* related third party imports
* local application/library specific imports

and also it would be even cleaner if you group the `from` to one block and `import` to other